docs/src/main/asciidoc/microprofile-metrics.adoc

```diff
@@ -21,6 +21,9 @@ Apart from application-specific metrics, which are described in this guide, you
 exposed by various Quarkus extensions. These are described in the guide for each particular extension that supports
 built-in metrics.
 
+IMPORTANT: link:micrometer[Micrometer] is the _opinionated_ approach to metrics for Quarkus.
```

Comment:
You changed another occurrence but not this one. Maybe it should be changed too?

Reply:
Thanks @gsmet, missed that one